modules/serverless-kafka-broker-configmap.adoc

```diff
@@ -56,6 +56,38 @@ data:
   bootstrap.servers: "my-cluster-kafka-bootstrap.kafka:9092"
 ----
 
+. To change templates for generating IDs and names used by your triggers, brokers, and channels, modify the `kafka-broker-config` config map:
```

Comment:
```suggestion
. To change templates for generating consumer group IDs and topic names used by your triggers, brokers, and channels, modify the `KnativeKafka` resource:
```

Reply:
Users need to configure the templates on the `KnativeKafka` resource in the `knative-eventing` namespace as the ConfigMap is managed by the operator:

```yaml
# ...
kind: KnativeKafka
metadata:
  name: <..>
  namespace: knative-eventing
# ...

spec:
  config:
    config-kafka-features:
       triggers-consumergroup-template: knative-trigger-{{ .Namespace }}-{{ .Name }}         # <-- change template value here
```